README.md

```diff
@@ -66,6 +66,7 @@ The stamp descriptor properties are made available on each stamp as `stamp.compo
 * `deepProperties` - A set of properties that will be added to new object instances by assignment with deep property merge.
 * `initializers` - A set of functions that will run in sequence. Stamp details and arguments get passed to initializers.
 * `staticProperties` - A set of static properties that will be copied by assignment to the stamp.
+* `deepStaticProperties` - A set of properties that will be added to the stamp by assignment with deep property merge.
```

Comment:
Looks good to me, may want to make it clear that these are **static** properties in the definition.